### PR TITLE
Preprod: movie: Beyond Skyline (2017) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
This film franchise is going downhill at a rate faster than Eliane's accelerated pregnancy. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2010s.st.txt [hunk 1]
```diff
@@ -2661,6 +2661,54 @@ monster [The fire demon Surtur.]
...
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+attack from outer space [The story follows a small group of people who become caught up in an alien invasion of Earth.]
+
+:: Major Themes
+father and son [The hard-boiled detective Mark Corley risked life and limb to save his son, Trent, who'd
+been taken captive by invading aliens aboard their mothership.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2010s.st.txt:2689]: Incorrect line break here - don't remember if my script would be able to repair it but either way, please take care not to leave linebreaks in the motivations.

File: notes/stories/film/film-scifi-2010s.st.txt [hunk 2]
```diff
@@ -2661,6 +2661,54 @@ monster [The fire demon Surtur.]
...
+extraterrestrial being [Alien invaders were going around abducting and/or killing people.]
+alien abduction [The invading aliens sucked any number of Los Angeles residents into their various spaceships by means of an ethereal blue light. In particular, Mark and his son Trent were taken aboard the alien mothership whereupon Trent has his brain removed and placed in the body of an alien bio-mechanical creature.]
+giant monster [Giant alien bio-mechanical creatures were rampaging around. The film culminated with a big fight between two of the giant creatures, one controlled by the alien leader, the other by Trent.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2010s.st.txt:2692]: Just a reflection: there are natural giant monsters like Godzilla, and artificial giant monsters that blur the line between robot and creature. A distinction worth making?

**Review by odinlake — CHANGES_REQUESTED** [review]
One syntax fix needed.

File: notes/stories/film/film-scifi-2010s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt]: Whoops. Fixed. While I'm still in the early stages of my investigation, we should be able to create GitHub Actions workflows to automatically catch these sorts of syntax errors.

File: notes/stories/film/film-scifi-2010s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt:2691]: I see where you're coming from, but lump them all together for now?

**Comment by paul-sheridan** [comment]
If you're satisfied with this film entry, let's try a "squash and merge" so that only the final version of the film entry is merged into master.

**Review by odinlake — APPROVED** [review]

**Pull request merged**